index.js

```diff
@@ -1,16 +1,28 @@
-module.exports = {
-  equals: 'fantasy-land/equals',
-  concat: 'fantasy-land/concat',
-  empty: 'fantasy-land/empty',
-  map: 'fantasy-land/map',
-  ap: 'fantasy-land/ap',
-  of: 'fantasy-land/of',
-  reduce: 'fantasy-land/reduce',
-  sequence: 'fantasy-land/sequence',
-  chain: 'fantasy-land/chain',
-  chainRec: 'fantasy-land/chainRec',
-  extend: 'fantasy-land/extend',
-  extract: 'fantasy-land/extract',
-  bimap: 'fantasy-land/bimap',
-  promap: 'fantasy-land/promap'
-}
+(function() {
+
+  'use strict';
+
+  var mapping = {
+    equals: 'fantasy-land/equals',
+    concat: 'fantasy-land/concat',
+    empty: 'fantasy-land/empty',
+    map: 'fantasy-land/map',
+    ap: 'fantasy-land/ap',
+    of: 'fantasy-land/of',
+    reduce: 'fantasy-land/reduce',
+    sequence: 'fantasy-land/sequence',
+    chain: 'fantasy-land/chain',
+    chainRec: 'fantasy-land/chainRec',
+    extend: 'fantasy-land/extend',
+    extract: 'fantasy-land/extract',
+    bimap: 'fantasy-land/bimap',
+    promap: 'fantasy-land/promap'
+  };
+
+  if (typeof module === 'object' && typeof module.exports === 'object') {
+    module.exports = mapping;
+  } else {
+    self.FantasyLand = mapping;
```

Comment:
See davidchambers/Base64.js#27. `self` works in more environments than `window` does.